src/Identity/Core/src/DefaultPasskeyHandler.cs

```diff
@@ -13,23 +14,154 @@ namespace Microsoft.AspNetCore.Identity;
 /// <summary>
 /// The default passkey handler.
 /// </summary>
-public partial class DefaultPasskeyHandler<TUser> : IPasskeyHandler<TUser>
+public sealed class DefaultPasskeyHandler<TUser> : IPasskeyHandler<TUser>
```

Comment:
This class was unsealed before because it had virtual methods that could be overridden for handling origin validation and attestation statement verification. However, that logic is now defined in callbacks in `PasskeyOptions`, and it no longer makes sense to extend this type. Therefore, I'm marking this class as `sealed` now.